src/services/player.rs

```diff
@@ -128,11 +127,30 @@ fn handle_message<M: Messenger>(
             let player = players
                 .get(&msg.local_conn_id)
                 .expect("Could not cross border: player not found");
+            messenger.broadcast_packet_remote(Packet::DestroyEntities(DestroyEntities {
+                entity_ids: vec![player.entity_id],
+            }));
```

Comment:
This is something that should have been in there before- the entity was supposed to be deleted as they cross the border so they wouldn't have two bodies. It appeared as though their original body was deleted, but that turned out to only be because the body walked out of bounds and disappeared in the eyes of others. We need it to actually be deleted now so that we can respawn the player when they walk back